src/fairseq2/models/wav2vec2/factory.py

```diff
@@ -76,7 +76,7 @@ class Wav2Vec2Config:
     temporal_mask_span_len: int = 10
     """The length of each temporal mask span that is applied over time steps."""
 
-    max_temporal_mask_prob: float = 0.65
+    max_temporal_mask_prob: float = 0.69
```

Comment:
Wondering if this change if just to use the default values from the paper or if there are other implications here.

Reply:
I checked the average masking rate with fairseq1, 0.69 gets us the closest to the hyperparam for wav2vec2 in fairseq1. This is just to match the default average masking rate in fairseq1 as specifid in the paper.

Reply:
Makes sense.